lib/PuppeteerSharp/Payload.cs

```diff
@@ -21,7 +21,8 @@ public class Payload
             "x-devtools-emulate-network-conditions-client-id",
             "cookie",
             "origin",
-            "content-type"
+            "content-type",
+            "intervention"

```

Comment:
The other day I kept thinking about the HashKey. Then I realized, Shouldn't a `const string[]` be even faster and lightweight?

Reply:
Hmm, I'm not sure.
You can run a benchmark to test that

Reply:
Not strong on this, just a thought